### DISCUSSION TITLE
Someone added wrong domain for my university, its someone's personal domain and is blocking access to many users

### DISCUSSION THREAD

**Discussion by lazydevpro** [community/community#64828] [GitHub Education] [answered]
### Hi everyone,

I got Github Education access as a student earlier and now I'm trying to renew it. Still, while trying to renew I found out that someone had given the wrong info or there was some technical issue because for my University "Indira Gandhi National Open University, Delhi" the email domain required is either - domnify.com - silomails.com, both are incorrect my university doesn't provide personal email to any student, one of the email leads to some person's personal website where he sells services and other domain seems to be not working.

I read in the email that I can mark ‘My selected school has incorrect or incomplete information e.g. domains or campus location’ but looks like that option is not available for renewal.

Now my question is even if I won't be able to renew mine how can I report this issue to fix the correct domain? because I believe a lot of students are missing out cause of this issue.

**Comment by arafajs** [comment, marked as answer]
You can report the issue by following these steps:

1. Go to the GitHub Education: https://education.github.com/contact page.
2. Select the "I have a question about my school's affiliation" option.
3. In the "Description" field, provide the following information:
4. Your university's name
5. The correct email domain for your university
6. A link to the personal website that you found when you tried to use the incorrect domain
7. Click on the "Submit" button.
